src/js/angular/clustermanagement/directives.js

```diff
@@ -338,7 +339,33 @@ clusterManagementDirectives.directive('clusterGraphicalView', ['$window', 'Local
                     nodes.forEach((node) => {
                         node.hostname = UriUtils.shortenIri(node.endpoint);
                         if (!_.isEmpty(node.recoveryStatus)) {
-                            const messageLabelKey = 'cluster_management.cluster_graphical_view.recovery_state.' + node.recoveryStatus.state.toLowerCase();
+                            let messageLabelKey = 'cluster_management.cluster_graphical_view.recovery_state.';
+                            const hasAffectedNodes = node.recoveryStatus.affectedNodes && node.recoveryStatus.affectedNodes.length > 0;
+                            const nodeRecoveryState = node.recoveryStatus.state;
+
+                            if (hasAffectedNodes) {
+                                switch (nodeRecoveryState) {
```

Comment:
Can you please replace this switch with the  more simple! Reorganize the label keys so that they allow simple resolution by concatenating `messageLabelKey + state.toLoverCase` without any additional conditional expressions.

Reply:
Done